docker/quoting-service/default.json

```diff
@@ -45,11 +50,10 @@
     "includeCauseExtension": false,
     "truncateExtensions": true
   },
-  "SIMPLE_ROUTING_MODE": true,
+  "SIMPLE_ROUTING_MODE": false,
```

Comment:
Reason ?

Reply:
@vijayg10 ^^

Reply:
This is for the integration tests and not the default value in config/default.json.
As part of this PR https://github.com/mojaloop/quoting-service/pull/346 integrations tests are hardened and need persistance in the quoting service.